test/powershell/Modules/Microsoft.PowerShell.Core/Out-Host.Tests.ps1

```diff
@@ -27,8 +21,9 @@ Describe "Out-Host Tests" -tag CI {
     }
     It "Out-Host writes to host output" {
         $stringToWrite = "thing to write"
+        $stringExpected = "::$($stringToWrite):NewLine"
```

Comment:
This is debug output - we can use any perfectly visible symbol.
The only restriction is that not to use it in the source debug string (<value>).
This character ':' is rarely used, so I suppose it is appropriate.
Or I missed something?